### PR TITLE
feat: add cdn package

### PR DESCRIPTION
**Pull request by enesozturk**
# Changes

Add the CDN package using the wagmi client for Web3Modal referenced from @glitch-txs's PoC: https://github.com/glitch-txs/web3modal-cdn

# Associated Issues

[Slack Thread](https://walletconnect.slack.com/archives/C03RVH94K5K/p1716012612378069)

### CODE REVIEW

File: packages/cdn/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  "peerDependenciesMeta": {
+    "typescript": {
+      "optional": true
+    }
+  },
+  "dependencies": {
+    "@wagmi/connectors": "^5.0.2",
+    "@wagmi/core": "^2.10.2",
+    "@web3modal/wagmi": "^4.2.1",
```
Comment (svenvoskamp) [inline packages/cdn/package.json:29]: Update dependencies

**Review by svenvoskamp — APPROVED** [review]

**Review by svenvoskamp — APPROVED** [review]

File: packages/cdn/vite.config.ts [hunk 2]
```diff
@@ -0,0 +1,21 @@
+import { defineConfig } from 'vite'
+import { nodePolyfills } from 'vite-plugin-node-polyfills'
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  build: {
+    lib: {
+      entry: 'index.ts',
+      name: 'cdn-wagmi',
+      fileName: 'cdn-wagmi'
```
Comment (glitch-txs) [inline packages/cdn/vite.config.ts:10]: We can change this names btw

Comment (enesozturk) [reply packages/cdn/vite.config.ts:10]: Totally

File: package.json [hunk 3]
```diff
@@ -5,7 +5,7 @@
   "type": "module",
   "scripts": {
     "build:clean": "turbo run build:clean",
-    "build": "turbo run build:clean; turbo run build",
+    "build": "turbo run build:clean; turbo run build --concurrency=30",
```
[the remark this answers is not in the document]
Comment (enesozturk) [reply package.json:8]: Loll one extra step got caught by Turbo and we needed to make it at least 30 (15 cleaning and 15 building)

**Review by glitch-txs — APPROVED** [review]

**Pull request merged**